src/AddTodoForm.js

```diff
@@ -24,8 +25,9 @@ function AddTodoForm({onAddTodo}) {
     return(
         <div>
             <form onSubmit={handleAddToDo} >
-                <label htmlFor="todoTitle">Title </label>
-                <input id="todoTitle" name="title" type="text" value={todoTitle} onChange={handleTitleChange}></input>
+               <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange} label={"Title"}> 
```

Comment:
You don't need to pass the `label `as a prop anymore because you use `children` property instead.